content/yaml-testing/detox-testing.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Detox (E2E) test automation
+description: How to run Detox tests with codemagic.yaml
+weight: 5
+
+---
+
+Detox testing refers to end-to-end (E2E) testing for React Native apps using the Detox framework. Detox provides a gray box testing solution, meaning it runs tests on a real device or emulator while also accessing the app's internal state for better synchronization.
+
+
+## Prerequisites
+
+1. **npm/yarn** are pre-installed
+2. A ready React Native project
```

Comment:
Maybe you should start the list with React Native project here? Also, what do you mean by ready? Maybe simply "A React Native project"